React-Homeworks/src/components/Footer/Copyright/Copyright.jsx

```diff
@@ -0,0 +1,15 @@
+import Logo from "../../Logo/Logo"
+import "./Copyright.css"
+
+function Copyright() {
+    return(
+        <div className="footer_copy flex-elem">
+            <Logo/>
+            <h2><p>Takeaway & Delivery template</p><p>for small - medium businesses.</p></h2>
```

Comment:
In case you need to organise 2 lines the better way is to wrap each of lines into span and make h2 tag as a flex container